### PR TITLE
Add helpers in documentation

### PR DESCRIPTION
**Pull request by soupette**
- Add helpers documentation in Plugin Development section

### CODE REVIEW

File: docs/3.x.x/en/plugin-development/utils.md [hunk 1]
```diff
@@ -2,6 +2,63 @@
 
 Strapi provides helpers so you don't have to develop again and again the same generic functions.
 
+## Auth
+
+`auth.js` lets you get, set and delete data in either the browser's `localStorage` or `sessionStorage`.
+
+### Methods
+
+| Name | Description |
```
Comment (pierreburgy) [inline docs/3.x.x/en/plugin-development/utils.md]: The table is not displayed properly: https://github.com/strapi/strapi/blob/add-documentation/docs/3.x.x/en/plugin-development/utils.md

File: docs/3.x.x/en/plugin-development/utils.md [hunk 2]
```diff
@@ -2,6 +2,63 @@
...
+| ---- | ––––––––––– |
```
Comment (lauriejim) [inline docs/3.x.x/en/plugin-development/utils.md]: Please replace `| ---- | ––––––––––– |` with `| ---- | ----------- |`

File: docs/3.x.x/en/plugin-development/utils.md [hunk 3]
```diff
@@ -2,6 +2,63 @@
...
+
+```js
+import { darken } from 'utils/colors';
+
+const linkColor = darken('#f5f5f5', 1.5); // Will darken #F5F5F5 by 1.5% which gives #f2f2f2.
+```
+
+## Get URL Query Parameters
+
+The helpers allows to retrieve the query parameters in the URL
```
Comment (pierreburgy) [inline docs/3.x.x/en/plugin-development/utils.md]: Please add a dot at the end of the sentence.

File: docs/3.x.x/en/plugin-development/utils.md [hunk 4]
```diff
@@ -2,6 +2,63 @@
...
+
+### Methods
+
+| Name | Description |
+| ---- | ––––––––––– |
+| clear(key) | Remove the data in either `localStorage` or `sessionStorage` |
+| clearAppStorage() | Remove all data from both storage |
+| clearToken() | Remove the user's `jwt Token` in the appropriate browser's storage |
+| clearUserInfo() | Remove the user's info from storage |
+| get(key) | Get the item in the browser's storage |
```
Comment (pierreburgy) [inline docs/3.x.x/en/plugin-development/utils.md]: Is it from the `sessionStorage` or from the `localStorage`?

File: docs/3.x.x/en/plugin-development/utils.md [hunk 5]
```diff
@@ -2,6 +2,63 @@
...
+| getToken() | Get the user's `jwtToken` |
+| getUserInfo() | Get the user's infos |
+| set(value, key) | Set an item in the `sessionStorage`. If `true` is passed a 3rd parameter it sets the value in the `localStorage` |
```
Comment (pierreburgy) [inline docs/3.x.x/en/plugin-development/utils.md]: Wouldn't it be better to move `get` and `set` functions in a util dedicated to storage (eg. `utils/storage`).

Comment (pierreburgy) [reply docs/3.x.x/en/plugin-development/utils.md]: Add `isLocalStorage` as parameter in this function.

File: docs/3.x.x/en/plugin-development/utils.md (hunk 4 above)
Comment (pierreburgy) [reply docs/3.x.x/en/plugin-development/utils.md]: Add `isLocalStorage` as parameter in this function.

File: docs/3.x.x/en/plugin-development/utils.md [hunk 6]
```diff
@@ -2,6 +2,63 @@
...
+| setToken(value) | Set the user's `jwtToken` in the `sessionStorage`. If `true` is passed a 3rd parameter it sets the value in the `localStorage` |
```
Comment (pierreburgy) [inline docs/3.x.x/en/plugin-development/utils.md]: Add `isLocalStorage` as parameter in this function.

File: docs/3.x.x/en/plugin-development/utils.md [hunk 7]
```diff
@@ -2,6 +2,63 @@
...
+| setUserInfo(value) | Set the user's info in the `sessionStorage`. If `true` is passed a 3rd parameter it sets the value in the `localStorage` |
```
Comment (pierreburgy) [inline docs/3.x.x/en/plugin-development/utils.md]: Add `isLocalStorage` as parameter in this function.

File: docs/3.x.x/en/plugin-development/utils.md (hunk 5 above)
Comment (soupette) [reply docs/3.x.x/en/plugin-development/utils.md]: You're right but it's not related  with this PR

File: docs/3.x.x/en/plugin-development/utils.md (hunk 4 above)
Comment (soupette) [reply docs/3.x.x/en/plugin-development/utils.md]: There's no parameter needed in this function. Here's its code :

```
get(key) {
    if (localStorage && localStorage.getItem(key)) {
      return parse(localStorage.getItem(key));
    }

    if (sessionStorage && sessionStorage.getItem(key)) {
      return parse(sessionStorage.getItem(key));
    }

    return null;
  },
```

**Pull request merged**